netscaler/client.go

```diff
@@ -50,7 +50,7 @@ type NitroClient struct {
 //Please use NewNitroClientFromParams
 func NewNitroClient(url string, username string, password string) *NitroClient {
 	c := new(NitroClient)
-	c.url = strings.Trim(url, " /") + "/nitro/v1/config/"
+	c.url = strings.Trim(url, " /") + "/nitro/v1/"
```

Comment:
Instead of adding this back in every function in netscaler_resource.go, add a new property to the client called 'statsUrl'.
So, in netscaler_resource.go, for listStats, you would use c.statsUrl + resourceType. All other changes you have made would then be unnecessary. I will change the name of c.url to c.configUrl after you resubmit your pull request